server/db/models/index.js

```diff
@@ -0,0 +1,31 @@
+const User = require('./users')
+const Contact = require('./contacts')
+const Order = require('./orders')
+/**
+ * If we had any associations to make, this would be a great place to put them!
+ * ex. if we had another model called BlogPost, we might say:
+ *
+ *    BlogPost.belongsTo(User)
+ */
+Contact.belongsTo(User)
+User.hasMany(Contact)
+Order.belongsTo(User) //order belongs to one user with userId
+User.hasMany(Order)
+User.afterCreate(user => {
+  //create the User's cart
+  Order.create({
+    dateOfSubmission:new Date(),
+    userId: user.id
+  })
+})
```

Comment:
Does this mean after the user has been created it automatically creates an order?